src/ProductConstructionService/ProductConstructionService.ServiceDefaults/MetricConsts.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace ProductConstructionService.ServiceDefaults;
+public static class MetricConsts
+{
+    public const string JobMeterName = "ProductConstructionService.Api";
+
+    public const string JobDurationMillisecondsHistogram = "duration";
```

Comment:
It's weird that this is here while the full metric name is in the recorder class. I'd unify this into one place, possibly the recorder?

Reply:
We need the `JobMeterName` in the `ServiceDefaults` project, since the Meter names need to match. I'll remove the other const tho, it really doesn't belong there